### PR TITLE
convert tensorflow protobuf to MPC graph

### PR DESCRIPTION
**Pull request by justin1121**
Closes https://github.com/mortendahl/tf-encrypted/issues/26

Here's some code that converts a protobuf tensorflow graph into code that just builds a mpc graph.

### CODE REVIEW

File: tensorflow_encrypted/convert/register.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import tensorflow as tf
+
+from typing import Any, Dict, List
+
+
+def register() -> Dict[str, Any]:
+    reg = {}
```
Comment (mortendahl) [inline tensorflow_encrypted/convert/register.py]: you could write this like 
```return {
  'Placeholder': placeholder,
  'Const': constant,
  ...
}```

File: examples/convert.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+def export_vgg16():
+    vgg16 = tf.keras.applications.VGG16()
+
+    sess = K.backend.get_session()
+
+    pred = [None]
+    pred_node_names = [""]
+    pred_node_names[0] = "output"
+    pred[0] = tf.identity(vgg16.outputs[0], name=pred_node_names[0])
```
Comment (morgangiraud) [inline examples/convert.py]: Why do you define `pred = [None]` and then reassign it directly `pred[0] = ...`?
Same question for `pred_node_names `

File: tensorflow_encrypted/convert/register.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        'Reshape': reshape,
+        'MatMul': matmul,
+        'Softmax': softmax,
+    }
+
+    return reg
+
+
+def placeholder(node: Any, inputs: List[str], output_lookup: Dict[str, Any]) -> Any:
+    return tf.placeholder(node.attr["dtype"].type,
```
Comment (morgangiraud) [inline tensorflow_encrypted/convert/register.py]: I guess the goal here is to replace those with `tfe` functions?

File: examples/convert.py (hunk 2 above)
Comment (justin1121) [reply examples/convert.py]: Not reason really. This code was borrowed from elsewhere and I haven't make it nice nice yet.

File: tensorflow_encrypted/convert/register.py (hunk 3 above)
Comment (justin1121) [reply tensorflow_encrypted/convert/register.py]: You got it!

**Comment by justin1121** [comment]
Yeah not sure how important those tests would be right now. I've started getting rid of the 1-to-1 mapping stuff and the final MPC graph will be wildly different from the input graph.

**Comment by mortendahl** [comment]
let me take a look later today!

**Comment by justin1121** [comment]
Sounds good! Sorry it got so big 😦

**Pull request closed without merging**